ReactWindows/ReactNative.Shared/Views/View/ReactViewControl.cs

```diff
@@ -1,24 +1,37 @@
-﻿using System;
+using System;
 
 #if WINDOWS_UWP
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Data;
 #else
 using System.Windows;
 using System.Windows.Controls;
 #endif
 
-namespace ReactNative.UIManager
+namespace ReactNative.Views.View
 {
     /// <summary>
-    /// Represents a Canvas with an optional Border inside.
+    /// Native control for React view.
     /// </summary>
-    public class BorderedCanvas : Canvas
+    public class ReactViewControl : UserControl
```

Comment:
>UserControl [](start = 36, length = 11)

A while ago, every View mapped to a Border nested in a Canvas.

To noticeably improve performance, things were changed so Borders got created lazily and most Views mapped to just a Canvas.

With this change, each View will map to a UserControl which contains a Canvas. Will this regress performance?

Reply:
Yes, this will likely regress performance. A UserControl actually already has a built in border, but it doesn't have a `CornerRadius` prop, so the performance is even worse than it was originally. We may want to override `View` and check for the `isTabStop` property to decide which views we convert to controls, but this prototype is reasonable for a functionality MVP.